sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/properties/utils/PropertyConditionUtil.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.implementation.properties.utils;
+
+import org.springframework.util.StringUtils;
+
+/**
+ * Utility class for property conditions.
+ */
+public final class PropertyConditionUtil {
+
+    private PropertyConditionUtil() {
+    }
+
+    /**
+     * Ensure the prefix format is valid.
+     * @param prefixAttr the input prefix value
+     * @return the result valid prefix
+     */
+    public static String getValidPrefix(String prefixAttr) {
```

Comment:
this method is really bad named and not accurate description.

The whole class should be reworked, discuss this offline with me